Tabloid/client/src/components/tag/Tag.js

```diff
@@ -1,8 +1,6 @@
 import React, { useContext } from "react";
-import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
-import { faTrashAlt, faPencilAlt } from '@fortawesome/free-solid-svg-icons';
-import { TagContext } from "../../providers/TagProvider";
-import { useNavigate } from "react-router-dom"
+
+
 
```

Comment:
These imports need to stay in the file in order for your pr to function. However this file was before you started the pr just change it back to that and this should be good.